README.md

```diff
@@ -28,20 +28,6 @@ Here's an example, using CFPB's regulation H.
 At the end, you will have new directories for `regulation`, `layer`,
 `diff`, and `notice` which would mirror the JSON files sent to the API.
 
-## Quick Start with Modified Documents
```

Comment:
Why is this section being removed? We certainly need to update the `build_from.py` command, but I think this is quite useful to have in the README.